std/algorithm/mutation.d

```diff
@@ -2419,3 +2470,13 @@ void uninitializedFill(Range, Value)(Range range, Value value)
         // Doesn't matter whether fill is initialized or not
         return fill(range, value);
 }
+
+///
+nothrow unittest
+{
+    import core.stdc.stdlib : malloc;
+
+    auto s = (cast(int*) malloc(5 * int.sizeof))[0 .. 5];
```

Comment:
This leaks.. As an example it's *e-hm* fine. But unittest should absolutely do `scope(exit) free(s.ptr);`